ports/stm32h5/README.md

```diff
@@ -0,0 +1,27 @@
+# TinyUF2 for STM32H5
+
+TinyUF2 occupies around 16Kb + 1KB for CF2 in flash. Since H5 has uniform sector size of 8KB, ideally we would have bootloader size set to 24KB. However, due to the fact that only H503 can protect/secure individual sector, the rest of H5 family (H52x, H56x) can only secure flash in 4-sector (group) unit. Therefore application should start at
```

Comment:
shrink bootloader size, since we got just over 15KB + 1KB for CF2 (config), we will need 24KB. Further optimize code + usb is possible to get it down under 15KB, but that would make it difficult to expand later e.g custom board with lcd/neopixel etc. So 24KB for H503 and 32KB for the rest

```
LINK _build/stm32h503_nucleo/tinyuf2-stm32h503_nucleo.elf
Memory region         Used Size  Region Size  %age Used
             RAM:        9560 B      32764 B     29.18%
           FLASH:         15 KB        23 KB     65.22%
          CONFIG:          0 GB         1 KB      0.00%
```